catalog/aat.jsonld

```diff
@@ -1,5 +1,5 @@
 {
-  "@context": "https://schema.org/",
+  "@context": "https://schema.org/docs/jsonldcontext.jsonld",
```

Comment:
Slightly silly but necessary because of https://github.com/schemaorg/schemaorg/issues/2578.